pkg/operator/controller/dns_status_test.go

```diff
@@ -14,6 +14,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+var (
+	maxUnavailable = intstr.FromInt(1)
```

Comment:
Is there a reason to declare `maxUnavailable` here instead of inside `makeDaemonSet`?  We usually tend towards making tests and helpers self-contained.

Reply:
It is used in multiple functions and I removed it as a parameter to makeDaemonSet, so I declared it globally.  It's used in TestDNSStatusConditions, TestComputeDNSDegradedCondition, TestComputeDNSProgressingCondition, and TestSkippingStatusUpdates. 

For consistency, removed an extra declaration of it in TestDNSStatusConditions.

Reply:
All right, I won't insist on changing it.